validate.sh

```diff
@@ -66,3 +66,22 @@ call_bastion "kubectl logs --tail 10 \$(kubectl get pods -oname -l app=hello)" \
 echo "step 4 of the validation passed."
 
 call_bastion "kubectl delete -f ./manifests/network-policy.yaml" &> /dev/null
+
+# Now we test applying network policy based on residing namespace
+# This will create a hello-apps namespace and restrict access to hello-server only from this namespace
+call_bastion "kubectl apply -f ./manifests/network-policy-namespaced.yaml" &> /dev/null
+
+# http clients running in default namespace should be failing now
+call_bastion "kubectl logs --tail 10 \
+ \$(kubectl get pods -oname -l app=not-hello)" | grep "$TIMED_OUT" \
+ &> /dev/null || exit 1
```

Comment:
Let’s echo that this failed. Rather than exit 1

Easier to debug

Reply:
We need to exit. If we don't it will keep going to next steps and mess things up.

Reply:
I agree we need to exist, but we need an error message

Reply:
done.